spring-cloud-dataflow-server-core/src/main/resources/META-INF/dataflow-server-defaults.yml

```diff
@@ -66,6 +66,9 @@ spring:
             url: "{repository}/org/springframework/cloud/spring-cloud-dataflow-shell/{version}/spring-cloud-dataflow-shell-{version}.jar"
             checksum-sha1-url: "{repository}/org/springframework/cloud/spring-cloud-dataflow-shell/{version}/spring-cloud-dataflow-shell-{version}.jar.sha1"
             checksum-sha256-url: "{repository}/org/springframework/cloud/spring-cloud-dataflow-shell/{version}/spring-cloud-dataflow-shell-{version}.jar.sha256"
+      task:
+        version: "@project.version@"
```

Comment:
I didn't mean to add a new property, instead just put @project.version@ in a url where you now have {version}.

Reply:
If you're wondering above `{version}` use in `version-info`, that's just a hack(in AboutController) to get predictable addresses for tile.